test/components/previews/selection_preview.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SelectionPreview < Lookbook::Preview
+  # The selection controller stores the values of checkboxes in local storage. The local storage key is the url.
+  # When a user toggles a checkbox, the value is updated in local storage.
+  def default; end
+
+  # The selection controller stores the values of checkboxes in local storage. The local storage key is the url.
+  # This example uses the selection controller within a table to select/deselect rows.
+  # When a user toggles a checkbox, the value is updated in local storage.
+  def within_a_table; end
+end
```

Comment:
**chore (blocking):** Please add a tests for these.  You will probably want to include a page refresh of some sort to ensure that the checkboxes are still selected on reload.

Reply:
This is a big miss. Please see 2ed1488b97b1d7024d1c38fa0b3ed54b874d14de.

Reply:
I do not know why you say this is a big mess, I think this is coming really well.  Thanks for making the preview go back to the page.